medcat/cdb.py

```diff
@@ -802,3 +802,113 @@ def calculate_hash(self):
         self._hash = hasher.hexdigest()
         logger.info("Found new CDB hash: %s", self._hash)
         return self._hash
+
+    @staticmethod
```

Comment:
Perhaps this could go in some utility module? The CDB module is already rather large and I wouldn't want to enlarge it further if we can avoid it.

Though that could also be somewhat problematic since this does explicitly deal the CDB so would be great to have it be a part of this module.

With that said, perhaps (probably not within the context of this PR, though) we want to move CDB to its own package to give more opportunity to separate responsibilities into different modules. We could always import the class in the dunder init module to retain backwards compatibility.

Reply:
I don't see an ideal home for it currently in `medcat.utils`. `CDBMaker` could be a good home for it, although that probably would require some cleaning up.